mitiq/benchmarks/randomized_benchmarking.py

```diff
@@ -3,8 +3,7 @@
 # module.
 # Original authors: Cirq developers: Xiao Mi, Dave Bacon, Craig Gidney,
 # Ping Yeh, Matthew Neely.
-# Code URL = ('https://github.com/quantumlib/Cirq/blob/main/cirq-core/cirq/
-# experiments/qubit_characterizations.py').
+# Code URL = ('https://github.com/quantumlib/Cirq/blob/main/cirq-core/cirq/experiments/qubit_characterizations.py').
```

Comment:
Unrelated to the PR's scope; it just annoyed me that I couldn't Ctrl+Click on that link from my IDE 😄